projects/igniteui-angular-elements/src/app/custom-strategy.ts

```diff
@@ -66,9 +66,16 @@ class IgxCustomNgElementStrategy extends ComponentNgElementStrategy {
         (this as any).componentRef = {};
 
         const toBeOrphanedChildren = Array.from(element.children).filter(x => !this._componentFactory.ngContentSelectors.some(sel => x.matches(sel)));
+        const contentChildren = Array.from(element.children).filter(x => this._componentFactory.ngContentSelectors.some(sel => x.matches(sel)));
```

Comment:
This might need some more refinement since `ngContentSelectors` will give all projected children and not all of those need to have a content query (e.g. the state component right now) which won't schedule any query updates and thus won't trigger the event at all.
Should be possible to further filter the children through their matching config, need the `component` or `provideAs` to match those against the `componentConfig.contentQueries[n].childType`-s so we know for sure a child is expected to run query update and leave the emit for that.

Reply:
Ah, further annoying bit can be the `descendants` bit on the query, might need to account for that as well